src/screens/Submissions.tsx

```diff
@@ -133,36 +139,45 @@ function Submissions() {
             <TableBody>
               {currentPageData.map((item: IFileSubmission) => (
                 <TableRow key={`table-row-${item.upload_id}`}>
-                  {" "}
                   {/* Todo: Update this to use a more appropriate id as key */}
-                  <TableDataCell
-                    size="sm"
-                    className="flex-justify-center checkbox-row">
-                    <Checkbox />
-                  </TableDataCell>
                   <TableDataCell className="text-left">
                     {item.filename}
                   </TableDataCell>
                   <TableDataCell size="sm">
                     <Pill
-                      label={item.status}
-                      shape="slot"
-                      outline={false}
-                      inverse={false}
-                      color={uploadStatusColor(item.status)}
+                      label={getStatusDisplayValuesById(item.status).label}
+                      color={getStatusDisplayValuesById(item.status).pillColor}
                     />
                   </TableDataCell>
                   <TableDataCell size="md">
                     {new Date(item.timestamp).toLocaleString()}
                   </TableDataCell>
                   <TableDataCell size="sm" className="details-row">
-                    <Icons.Dots />
+                    {/* Todo: Instead of using a span here, we should update Icons to allow an onClick */}
```

Comment:
This comment can come out since we have a button here now. Good update

Reply:
Removed